wger/manager/tests/test_weight_log.py

```diff
@@ -254,7 +256,7 @@ def test_get_workout_session(self):
         workout2 = Workout.objects.get(pk=2)
 
         WorkoutLog.objects.all().delete()
-        l = WorkoutLog() # noqa
+        l = WorkoutLog()  # noqa
```

Comment:
I think the `#noqa` might not be needed here. Try remove it and run `flake8`, see what happens.

Reply:
It is needed to suppress the use of 'l' as a variable name.